docker/base.dockerfile

```diff
@@ -34,4 +34,4 @@ ENV RAILS_ENV=production
 
 # RUN rake assets:precompile
 
-RUN pip3 install youtube-dl
+RUN pip3 install -U youtube-dl
```

Comment:
Since youtube-dl is not installed in the image up to this point there's nothing to upgrade, so I don't think this will make a difference. The issue is that the image doesn't get rebuilt regularly, so of course an older version is installed. I created issue #39 to remember to work on this.